app/controllers/registrations_controller.rb

```diff
@@ -0,0 +1,12 @@
+class RegistrationsController < Devise::RegistrationsController
+
+
+    private
+    def sign_up_params
+        params.require(:user).permit(:name, :email, :password, :password_confirmation)
+    end
+
+    def update_params
+        params.require(:user).permit(:name, :email, :password, :password_confirmation)
+    end
```

Comment:
## Implent user sign out feature
- [ ] Now that our user can sign up, we don't have a way of them logging out. Create a method to enable this feature. Make sure  I as User I can successfully sign_out ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/sign_out_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/authentication))